plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/AppEngineDeployPreferencesPanel.java

```diff
@@ -596,6 +596,10 @@ protected IStatus validate() {
           return ValidationStatus.error(Messages.getString("projectselector.project.not.selected")); //$NON-NLS-1$
         }
       }
+      if (selectedProject != null && project != null) {
+        return ValidationStatus.info(Messages.getString("deploy.project.summary", project.getName(),
```

Comment:
Just to double-check: I'd like to make sure returning `info()` as `IStatus` is equivalent to `ok()` for the validation purposes and `info()` will enable the _Deploy_ button so that users can proceed to deploy.

Reply:
```
com.google.cloud.tools.eclipse.appengine.deploy.ui.DeployPreferencesDialog.createDialogArea(...).new ValidationMessageProvider() {...}.getMessageType(ValidationStatusProvider)

          public int getMessageType(ValidationStatusProvider statusProvider) {
            int type = super.getMessageType(statusProvider);
            setValid(type != IMessageProvider.ERROR);
            return type;
          }

```
 setting info status won't disable the   _Deploy_ button